### DISCUSSION TITLE
What is the significance of Priority in Quartz Triggers table?

### DISCUSSION THREAD

**Discussion by viveknuna** [quartznet/quartznet#2287] [Q&A] [answered]
I know the default value is 5. Some of my jobs should have the highest priority, they should have precedence if there is a clash. so what value shall I set for them? smaller than 5 or greater than 5?

Or shall I change the value for the low-priority jobs?

In terms of the number of schedules, low-priority jobs are 99% of total jobs.

@lahma

**Comment by jafin** [comment, marked as answer]
@viveknuna  https://www.quartz-scheduler.net/documentation/quartz-3.x/tutorial/more-about-triggers.html#common-trigger-attributes there is a section on priorities

>If you do not set a priority on a Trigger, then it will use the default priority of 5. Any integer value is allowed for priority, positive or negative. A larger number indicates a higher priority. i.e. A trigger with a Priority of 7 will have priority over trigger with a value of 5.

**Reply by viveknuna** [reply]
@jafin thank you, it answers my question
